### ISSUE TITLE
Citation online resources are not displayed

### ISSUE THREAD

**Issue by florianesser** [ISO-TC211/iso-geodetic-registry#73]
Asked by @maccraymer (item 5c):

> Online resources defined in the Excel spreadsheet (sheet `Source_Citation(CI#)`, column `N`) are not displayed in the Registry GUI. Are they in the Registry database? May not be in Registry output/GUI form

**Comment by florianesser** [comment]
@maccraymer The column `N` that contains the Citation online resources is currently ignored by the Excel importer. I think the reason is that there is not suitable attribute in ISO 19139's `CI_Citation` type where this information can be stored (see the definition of `CI_Citation_Type` in the [XML schema definition](https://schemas.isotc211.org/schemas/19139/-/gmd/1.0/citation.xsd)).

It might make sense to add an appropriate attribute to the Citation item class that needs to be created for [managing citations as items](https://github.com/ISO-TC211/iso-geodetic-registry/issues/50).

**Comment by maccraymer** [comment]
I agree that a Citation item class needs to be created with all columns in the spreadsheet as attributes. However, if this cannot be created now, please import column N and append it to information in column M (Other Details) so it is at least in the Registry and displayed.
